doc/neps/nep-0038-SIMD-optimizations.rst

```diff
@@ -180,20 +197,46 @@ Anyone writing a SIMD loop will use the ``npyv_load_u32`` macro instead of the
 architecture specific intrinsic. The code also supplies guard macros for
 compilation and runtime, so that the proper loops can be chosen.
 
+Two new build options are available to ``runtests.py`` and ``setup.py build``,
+which pass them on to be used in ``build_ext`` and ``build_clib``.
```

Comment:
This phrasing seems to suggest that if one calls `build_ext` directly, the options are _not_ passed on and will hence be missing. I think that's not the case (?), so a small tweak in phrasing would be nice.

Reply:
using the "less is more" principle, I simplified this.

Reply:
LGTM, thanks